nni/tools/nnictl/config_utils.py

```diff
@@ -4,64 +4,106 @@
 import os
 import json
 import shutil
+import sqlite3
 import time
-from .constants import NNICTL_HOME_DIR
+from .constants import NNI_HOME_DIR
 from .command_utils import print_error
 from .common_utils import get_file_lock
 
+def config_v0_to_v1(config: dict) -> dict:
```

Comment:
What does `v0` mean?

Reply:
v0 means the config format in nni.sqlite (clusterMetaData), will be deprecated soon.